client-java/controller-api/src/main/java/org/evomaster/client/java/controller/api/dto/database/schema/ColumnDto.java

```diff
@@ -23,10 +23,16 @@ public class ColumnDto {
     public boolean isUnsigned = false;
 
     public boolean isEnumeratedType = false;
+
+    public boolean isCompositeType = false;
```

Comment:
can add some comments on what a composite type is

Reply:
I added an extract of the postgres documentation